iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -1111,11 +1127,12 @@ def indexing_dims(self) -> list[IndexExpr]:
     @property
     def index(self) -> dict[IndexSymbol, IndexSequence]:
         custom = get_custom(self.value)
-        if custom.index is None:
+        custom_index = custom.index
```

Comment:
it saves up compute time! 😁

Reply:
since Reduction/GetResult index is a "property" method, I thought it'd be nice to not need to re-compute it after the first time round. let me know if you'd prefer it to be `custom.index` for better legibility! :)

Reply:
Makes sense, good to save compute time!